BaseFilter/sections/specific.txt

```diff
@@ -16787,9 +16787,9 @@ outlook.live.com#?##app > div[class] > div[class] > div[class] > div[class]:not(
 outlook.live.com#?#.customScrollBar[data-is-scrollable] > div[class] > div[class]:not([data-convid]):has(> div:not([id]):not([class]) > div[class] > div[class] > div[class] > div[class] > div[class] > .fbAdLink)
 !
 ! check-host.net
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢)$/):upward(2)
-/^https:\/\/check-host\.net\/images\/[0-9]?[0-9]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
-check-host.net##a[href*="extgo"]
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢|х)$/):upward(2)
+/^https:\/\/check-host\.net\/images\/[a-z]?[0-9]?[0-9]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div:has(> a[href]:contains(/^aeza.net|100%|Offshore/))
```

Comment:
Didn't see `aeza.net`. Is it still needed?
If so, also slightly change the containing part as follows `/^aeza\.net|100%|Offshore/`.

Reply:
`<div class=" relative nrkzbcharzcont" id="nrkzbchadaeza-permnnt">
<div class="mb-1 bg-gray-200 py-0.5 pl-1 border-l-8 border-[#bf72c6]">
The incredible is real! Profitable VPS 5.7 GHz, 6 locations, and more: <a href="[https://go.aeza.net/checkhost_en](view-source:https://go.aeza.net/checkhost_en)">aeza.net</a>
</div>
<div class="akda_z axz right-0">&#1093;</b></div>
</div>`

Reply:
I made the rule for blocking the ad in the top right, but since the rule could also block the Aeza one, I figured I might as well block it too.